### PR TITLE
Create the <Sunshine /> Component

### PR DESCRIPTION
**Pull request by cristicretu**
This PR solves issue [#10](https://github.com/Landmarks-Tech/landmarks-starter/issues/10)

Implemented a set of 20 Sun positions, each assigned to a section from the circle.
To change the position, pass a ```orientation``` prop, this will assign the Sun ranging from 0..19, and create a yellow gradient with the given direction.

Everything that will be under the Sunshine should be passed as a ```children prop```, as we need to use a zIndex.

As a ```tooltip```, it uses a newly created <Tooltip /> Component. Under the hood, it uses [Radix's Tooltip Primitive](https://www.radix-ui.com/docs/primitives/components/tooltip). The item that should be hoverable, (in our case the Sun Svg) needs to be passed as a ```children prop```, and the title of the Tooltip should be given, or, if it's left untouched, it will default to 'Sun position at noon'.

For reference, check the [Figjam for the component](https://www.figma.com/file/snBbCy8maRD38XMWQKX4KJ/Sunshine?node-id=0%3A1), where I documented how I found the Sun position.

### CODE REVIEW

**Comment by cristicretu** [comment]
Please make a commit so Vercel would deploy a preview @ancashoria.

File: components/Sunshine/Sunshine.tsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+      <Box
+        position="relative"
+        style={{
+          width: '100%',
+          height: '100%',
+          background: `linear-gradient(${orientation}deg, rgba(237,206,126, 0.5) 0%, rgba(237,206,126, 0) 30%)`
+        }}
+        {...rest}
+      >
+        <Tooltip content={tooltip} sideOffset={-5}>
```
Comment (ancashoria) [inline components/Sunshine/Sunshine.tsx:167]: The sun svg has some padding to it, with sideOffset we move the tooltip closer to negate it

File: components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  content: any
+  [key: string]: any
+}
+
+export function Tooltip({ children, content, className, ...rest }: IProps) {
+  return (
+    <TooltipPrimitives.Root>
+      <TooltipPrimitives.Trigger asChild>{children}</TooltipPrimitives.Trigger>
+      {/* forwarding all sprinkles props to Content */}
+      <Box component={TooltipPrimitives.Content} className={cn(styles.content, className)} {...rest}>
```
Comment (ancashoria) [inline components/Tooltip/Tooltip.tsx:19]: All sprinkles props received from call site are forwarded here

File: components/Tooltip/styles.css.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    paddingY: 'small',
+    color: 'neutral_5',
+    fontSize: '-1x',
+    background: 'white',
+    borderRadius: 'small',
+  }),
+  {
+    boxShadow:
+      'hsl(206 22% 7% / 35%) 0px 10px 38px -10px, hsl(206 22% 7% / 20%) 0px 10px 20px -15px',
+  }
```
Comment (ancashoria) [inline components/Tooltip/styles.css.ts:20]: `sprinkles` can be used as a function from `.css.ts` files. Notice `style()` receives an array that will combine sprinkles with regular vanilla-css. Using sprinkles whenever we can keeps the generated css low.

**Review by ancashoria — APPROVED** [review]

**Pull request merged**